Source/CarthageKit/Errors.swift

```diff
@@ -71,6 +72,10 @@ public enum CarthageError {
 	/// them.
 	case NoSharedSchemes(ProjectLocator)
 
+	/// The private cartfile contains dependencies that are already present in
+	/// the main cartfile.
+	case DuplicateDependenciesInPrivateCartfile([Dependency<VersionSpecifier>])
```

Comment:
Since it's not clear which side the version specifier represents, maybe we should just use a list of `ProjectIdentifier`s here.